tapa/graphir_conversion/utils.py

```diff
@@ -616,7 +567,7 @@ def get_fsm_def(
         content = f.read()
     module = Module(
         (fsm_file,),
-        True,
+        is_trimming_enabled=True,
         name=fsm_name,
```

Comment:
`name` should not be specified if `files` is specified (`name` is ignored if `files` is not empty)

```py
        (fsm_file,),
        is_trimming_enabled=True,
```